samples/subsys/display/lvgl/README.rst

```diff
@@ -49,17 +49,20 @@ Building and Running
 Example building for :ref:`nrf52840dk_nrf52840`:
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/display/lvgl
+   :zephyr-app: samples/subsys/display/lvgl
    :board: nrf52840dk_nrf52840
    :shield: adafruit_2_8_tft_touch_v2
    :goals: build flash
 
 Example building for :ref:`native_posix`:
 
-.. zephyr-app-commands::
-   :zephyr-app: samples/display/lvgl
-   :board: native_posix
-   :goals: build flash
```

Comment:
I suggest to keep it, update zephyr-app, and change goals to `:goals: build run`, 
in the docu it will look like this:
```
west build -b native_posix samples/subsys/display/lvgl
west build -t run
```

Reply:
Thanks for the suggestion. Updated.

Reply:
Have a few changes been lost?

Reply:
Yes. For some reason, the above excerpt was not tagged as outdated after the fixing push.